babbage.nomad

```diff
@@ -18,6 +18,12 @@ job "babbage" {
   group "web" {
     count = "{{WEB_TASK_COUNT}}"
 
+    spread {
+      attribute = "${node.class}"
```

Comment:
```suggestion
      attribute = "${node.unique.id}"
```
The examples need an `attribute` that varies, but `node.class` is constrained (later) to `web-mount`, so doesn't vary.  Therefore, I think this needs to be the (varying) host IDs on which it will be run

Reply:
review feedback applied